opendevin/runtime/client/client.py

```diff
@@ -61,55 +62,102 @@ def __init__(self, plugins_to_load: list[Plugin], work_dir: str) -> None:
 
     def _init_bash_shell(self, work_dir: str) -> None:
         self.shell = pexpect.spawn('/bin/bash', encoding='utf-8', echo=False)
-        self.__bash_PS1 = r'[PEXPECT_BEGIN] \u@\h:\w [PEXPECT_END]'
-
-        # This should NOT match "PS1=\u@\h:\w [PEXPECT]$" when `env` is executed
-        self.__bash_expect_regex = (
-            r'\[PEXPECT_BEGIN\] ([a-z0-9_-]*)@([a-zA-Z0-9.-]*):(.+) \[PEXPECT_END\]'
```

Comment:
There was a reason why i use `[a-z0-9_-]*)@([a-zA-Z0-9.-]*):(.+)` - because other wise, when agent runs `env`, it will be stopped by the `PS1=` env var printed by the `env` command, causing the parsing to be broken.

Reply:
Yes! I think this got caught in the test cases - i added that one to make sure it is not broken :D

Reply:
Hmm all tests in `test_runtime_client.py` pass in my local WSL, though

Reply:
Imo, there is an issue with using the ServerRuntime class in the test setup as it doesn't seem to correctly pull in `SANDBOX_ENV_` vars like the `EventStreamRuntime`does.